util/src/windows/scoped_handle.cc

```diff
@@ -0,0 +1,17 @@
+#include <leatherman/util/windows/scoped_handle.hpp>
+#include <windows.h>
+
+using namespace std;
+
+namespace leatherman { namespace util { namespace windows {
+
+    scoped_handle::scoped_handle(HANDLE h) :
+        scoped_resource(h, CloseHandle)
```

Comment:
It'd be nice if we passed a member function that checked for `INVALID_HANDLE` to avoid needless calls to `Close` with an invalid handle (would probably cause warnings under Application Verifier if we ran it).